api/bertymessenger.proto

```diff
@@ -13,6 +13,9 @@ service MessengerService {
   // InstanceShareableBertyID returns a Berty ID that can be shared as a string, QR code or deep link.
   rpc InstanceShareableBertyID(InstanceShareableBertyID.Request) returns (InstanceShareableBertyID.Reply);
 
+  // InstanceShareableBertyGroup returns a Berty Group that can be shared as a string, QR code or deep link.
+  rpc InstanceShareableBertyGroup(InstanceShareableBertyGroup.Request) returns (InstanceShareableBertyGroup.Reply);
```

Comment:
Not sure about the naming here, the first one was about sharing the berty ID of the current instance

Reply:
Ok, renaming to `ShareableBertyGroup`